source/extensions/filters/http/jwt_authn/extractor.cc

```diff
@@ -211,6 +216,36 @@ std::vector<JwtLocationConstPtr> ExtractorImpl::extract(const Http::HeaderMap& h
   return tokens;
 }
 
+// as specified in RFC-4648 § 5, plus dot (period, 0x2e), of which two are required in the JWT
+constexpr char ConstantBase64UrlEncodingCharsPlusDot[] =
+    "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_.";
+
+// Returns a token, not a URL: skips non-Base64Url-legal (or dot) characters, collects following
+// Base64Url+dot string until first non-Base64Url char.
+//
+// The input parameters:
+//    "value_str" - the header value string, perhaps "Bearer string....", and
+//    "after" - the offset into that string after which to begin looking for JWT-legal characters
+//
+// For backwards compatibility, if it finds no suitable string, it returns value_str as-is.
+//
+// It is forgiving w.r.t. dots/periods, as the exact syntax will be verified after extraction.
+//
+// See RFC-7519 § 2, RFC-7515 § 2, and RFC-4648 "Base-N Encodings" § 5.
+absl::string_view ExtractorImpl::extractJWT(const absl::string_view& value_str,
```

Comment:
pass string_view by value instead of reference

Reply:
Done.

FWIW, I was following the pattern of the other methods in the class, which take const references. (And I thought the overhead of copying the absl::string_view would be a (minor) problem.)